README.md

```diff
@@ -62,3 +62,40 @@ The metrics exposed by this Exporter are the following.
 | `opensearch_dashboards_resp_time_max`          | Maximum response time in milliseconds                                                                                           | Gauge |
 | `opensearch_dashboards_req_disconnects`        | Request disconnections count                                                                                                    | Gauge |
 | `opensearch_dashboards_req_total`              | Total request count                                                                                                             | Gauge |
+
+
+## Debian Package
+This project can build a debian package that will be used on [opensearch-dashboards-snap](https://github.com/canonical/opensearch-dashboards-snap). To manually build and release in a ppa, do the following steps:
+
+1. Create a new version in the debian changelog:
+
+```shell
+dch -v <version> "<MSG>"
+```
+After that, go to the `changelog` file and set the urgency and ubuntu release:
+
+```
+prometheus-opensearch-dashboards-exporter <version> noble; urgency=medium
+
+  * My change
+
+ -- Joe Doe <joe.doe@foo.com>  Thu, 16 Jan 2025 18:25:59 -0300
+```
+
+2. Build the package signing with a [registered launchpad GPG key](https://launchpad.net/+help-registry/openpgp-keys.html)
+
+You can get the number of your GPG key by running:
+
+```shell
+gpg --list-keys
+```
+
+```shell
+debuild -S -sa -k<KEY_ID>
```

Comment:
can we add these steps as part of the release pipeline?

Reply:
I think it's possible. I found a canonical project that does that:
https://github.com/canonical/mir/blob/main/.github/workflows/ppa-upload.yml

However, I don't feel that is nice to use those third parties actions for GPG key or to Set up Launchpad credentials.

I didn't find so far a 100% canonical action to do it.

Reply:
Understood, that's okay, we can explore separately